schema/examples/portal/manifest-signer-without-pin.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<portal-signature-job-manifest xmlns="http://signering.posten.no/schema/v1">
+    <signers>
+        <signer>
+            <link-notification>
+                <email address="email@hotmail.com"/>
```

Comment:
Personlig foretrekker jeg å bruke `@example.com` som e-postadresser i eksempler, siden dette domenet eksisterer for [nøyaktig dette formålet](https://tools.ietf.org/html/rfc2606#section-3).